grafana_client/knowledge.py

```diff
@@ -285,7 +285,35 @@ def query_factory(datasource, model: Optional[dict] = None, expression: Optional
                 "default": None,
             },
         ]
+    elif datasource_type == "grafana-redshift-datasource":
+        query = {
+            "datasource": {
+                "type": datasource["type"],
+                "uid": datasource.get("uid"),
+            },
+            "datasourceId": datasource.get("id"),
+            "rawSQL": expression,
+        }
 
+        attrs = [
+            {
+                "name": "format",
+                "default": 0,
+                "choices": [0, 1],
+            },
```

Comment:
Should the choice of response format be obtained from the user in one way or another? Is it fine to leave for a subsequent iteration?